sdks/python/apache_beam/io/textio.py

```diff
@@ -175,6 +182,33 @@ def split_points_unclaimed(stop_position):
         if num_bytes_to_next_record < 0:
           break
 
+  def _process_header(self, file_to_read, read_buffer):
+    # Returns a tuple containing the position in file after processing header
+    # records and a list of decoded header lines that match
+    # 'header_matcher_predicate'.
+    header_lines = []
+    position = self._skip_lines(
+        file_to_read, read_buffer,
+        self._skip_header_lines) if self._skip_header_lines else 0
+    while self._header_matcher_predicate:
```

Comment:
<!--new_thread; commit:70ade9d4fe6a8e31ff0be4d39b731510c93b4bf9; resolved:0-->
Please make this to "if self._header_matcher_predicate:" followed by a "while True" to make the code more readable.